### DISCUSSION TITLE
How can I disable completion of maven dependencies, especially in Java editor ?

### DISCUSSION THREAD

**Discussion by glhez** [eclipse-m2e/m2e-core#1656] [Q&A] [answered]
Hello,

I would like to know how I could disable the "maven completion" that display when you ask for a quick fix in such case : 

![image](https://github.com/eclipse-m2e/m2e-core/assets/10855953/87603991-dd16-4f09-9e6c-86752d352296)

In this case, I am writing :

```
@Test void test() {
   is = mock(InputStream.class);
}
```

The code is obviously wrong but I'm accustomed to let Eclipse "complete" the field/variable declaration: in this case, I would have wanted to select "Create locale variable".

However, from time to time, I select the Maven "Add [some GAV]]" option by mistake - needless to say this fire an hell of dependency update and annoying project build (I do understand that updating one dependency would trigger such build, ... but in my case this has a tendency to block Eclipse)

I would like to remove them from any Java editor or at least make sure they go last.

How can I do that ?

Is that present as an option here ?

![image](https://github.com/eclipse-m2e/m2e-core/assets/10855953/73c29b8e-7c89-4311-abcc-6d57a44751ad)

**Comment by laeubi** [comment, marked as answer]
![grafik](https://github.com/eclipse-m2e/m2e-core/assets/1331477/14872d35-1b5d-4d9a-95c6-db96a7a6e62a)

**Reply by glhez** [reply]
Thanks !
